code/modules/mob/dead/new_player/sprite_accessories/ipc.dm

```diff
@@ -232,12 +232,71 @@
 	name = "Right Angle"
 	icon_state = "rangle"
 
+/datum/sprite_accessory/ipc_antennas/sprinter
+	name = "Sprinter Tail"
+	icon_state = "sprinter"
+
+/datum/sprite_accessory/ipc_antennas/simple
+	name = "Synth Simple"
+	icon_state = "simple_synth"
+
+/datum/sprite_accessory/ipc_antennas/short
+	name = "Synth Short"
+	icon_state = "short_synth"
+
+/datum/sprite_accessory/ipc_antennas/curled
+	name = "Synth Curled"
+	icon_state = "curled_synth"
+
+/datum/sprite_accessory/ipc_antennas/ram //remade
+	name = "Synth Ram"
+	icon_state = "ram_synth"
+
+// Start tails
+
+/datum/sprite_accessory/ipc_tail
+	icon = 'icons/mob/ipc_accessories.dmi'
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/ipc_tail/none
+	name = "None"
+	icon_state = "none"
+
+/datum/sprite_accessory/ipc_tail/lizard
+	name = "Synthetic Sarathi"
+	icon_state = "synth"
+
+/datum/sprite_accessory/ipc_tail/lizard_big
+	name = "Synthetic Sarathi Large"
+	icon_state = "large"
+
+/datum/sprite_accessory/ipc_tail/plug
+	name = "Power Plug"
+	icon_state = "plug"
+	secondary_color = TRUE
+
+/datum/sprite_accessory/ipc_tail/cat
+	name = "Pawsitrons Cat"
+	icon_state = "cat"
+
+/datum/sprite_accessory/ipc_tail/fox
+	name = "Pawsitrons Fox"
```

Comment:
Don't like these, doesn't feel very ipc to me

Reply:
why not? besides,synths aren't very classic ipc-ey,either

Reply:
I love them and I've seen a couple IPCs with them. They're very cool